Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -435,6 +438,49 @@ private void UpdateHeldItems(EntityUid uid, EntityUid handcuff, CuffableComponen
                 EnsureComp<UnremoveableComponent>(virtItem2.Value);
         }
 
+        /// <summary>
+        /// Checks if the handcuff is stackable, and creates a new handcuff entity if the stack requires it.                                   
+        /// </summary>
+        /// <returns></returns>
+        public bool TrySpawnCuffSplitStack(EntityUid handcuff, EntityUid user, EntityUid target, [NotNullWhen(true)] out EntityUid? handcuffsplit)
+        {
+            if (!HasComp<HandcuffComponent>(handcuff))
+            {
+                handcuffsplit = null;
+                return false;
+            }
+            if (TryComp<StackComponent>(handcuff, out var stackComp))
+            {
+                if (_stacks.GetCount(handcuff, stackComp) >= 1)
+                {
+                    _stacks.Use(handcuff, 1, stackComp);
+
+                    if (_net.IsServer) /// let the server spawn because client mispredicts
+                    {
+                        var pos = Transform(target).Coordinates;
+                        handcuffsplit = Spawn("Zipties", pos); /// This should somehow get the proto ID instead of zipties, but fuck if I know how.
+                        return true;
+                    }
+                    else
+                    {
+                        handcuffsplit = null;
+                        return false;
+                    }
+                }
+                else
+                {
+                    handcuffsplit = null;
+                    return false;
+                }
+            }
+            else
+            {
+                handcuffsplit = handcuff;
+                _hands.TryDrop(user, handcuff);
+                return true;
+            }
```

Comment:
```suggestion
            if (!TryComp<StackComponent>(handcuff, out var stackComp))
            {
                handcuffsplit = handcuff;
                _hands.TryDrop(user, handcuff);
                return true;
            }

            if (_stacks.GetCount(handcuff, stackComp) >= 1)
            {
                _stacks.Use(handcuff, 1, stackComp);

                var pos = Transform(target).Coordinates;
                handcuffsplit = _net.IsServer ? Spawn("Zipties", pos) : null;
            }
            else
                handcuffsplit = null;

            return handcuffsplit != null;
```

I could compact this further but you can probably tell why not below :)
```cs
// ...
var pos = Transform(target).Coordinates;
handcuffsplit = _stacks.GetCount(handcuff, stackComp >=1) ? : _net.IsServer ? : Spawn("Zipties", pos) : null : null;
if (handcuffsplit != null)
    _stacks.Use(handcuff, 1, stackComp);
return handcuffsplit != null;
```